Reordering modifiers, clean up imports, and deleting commented-out code

ullenius opened the pull request:
Minor clean up. Does **3** things:

- Removing commented out lines of code.

> "It makes me crazy to see stretches of code that are commented out. 
> Who knows how old it is?
> Who knows whether or not it's meaningful? 
> Yet no one will delete it because everyone assumes someone else needs it or has plans for it. Commented-out code is an abomination." 

-Clean Code (chapter 17) by [Uncle Bob](https://en.wikipedia.org/wiki/Robert_C._Martin)

- Cleaning up unused imports and deleting imports from the same package.

- Reordering modifiers to comply with the Java Language Specification:

> The Java Language Specification recommends listing modifiers in the following order:
> 
> 1. Annotations
> 2. public
> 3. protected
> 4. private
> 5. abstract
> 6. static
> 7. final
> 8. transient
> 9. volatile
> 10. synchronized
> 11. native
> 12. strictfp

stefanbirkner commented:
The change in `FailOnTimeoutOutTest` is already part of 1254795902e56eb545e1fe696827924bc5e82b66. Removing it from the commits in this PR will solve the conflict. I can do this when merging the PR.

kcooney commented on src/main/java/junit/extensions/ActiveTestSuite.java:
```diff
@@ -43,7 +43,6 @@ public void runTest(final Test test, final TestResult result) {
             public void run() {
                 try {
                     // inlined due to limitation in VA/Java
-                    //ActiveTestSuite.super.runTest(test, result);
```
I think we should add this comment back, since i clarifies what was inlined.

ullenius replied:
Fixed in 460e488

kcooney commented on src/main/java/junit/extensions/ActiveTestSuite.java:
```diff
@@ -1,9 +1,6 @@
-import junit.framework.TestCase;
-import junit.framework.TestResult;
-import junit.framework.TestSuite;
+import junit.framework.*;
```
I prefer we avoid using globs in imports. I wouldn't go so far as to suggest to remove existing globs, but let's not add more.

ullenius replied:
Reverted in 7b53777.

marcphilipp commented:
@ullenius Could you please rebase on `main` to resolve the conflict?

ullenius commented:
> @ullenius Could you please rebase on `main` to resolve the conflict?

No problem. I removed `FailOnTimeoutOutTest` from the PR as @stefanbirkner suggested and rebased.

stefanbirkner commented:
I'm merging the PR.
